Meta/check-newlines-at-eof.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+script_path=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+cd "$script_path/.." || exit 1
+
+MISSING_NEWLINE_AT_EOF_ERRORS=()
+MORE_THAN_ONE_NEWLINE_AT_EOF_ERRORS=()
+
+while IFS= read -r f; do
+    [ -s "$f" ] || continue
+
+    if [ "$(tail -n 1 "$f" | wc -l)" != 1 ]; then
+        MISSING_NEWLINE_AT_EOF_ERRORS+=( "$f" )
+    elif [[ "$(tail -n 2 "$f")" =~ ^[[:space:]]*$ ]]; then
```

Comment:
```suggestion
    elif [[ "$(tail -n 1 "$f")" =~ ^[[:space:]]*$ ]]; then
```

This should be a 1, not a 2!
I need to fix more files.
```
Files that have blank lines at the end: Applications/TextEditor/TextEditorWindow.gml Meta/text-to-cpp-string.sh Ports/indent/package.sh Shell/Tests/backgrounding.sh
```

Reply:
https://github.com/SerenityOS/serenity/runs/1639805371